packages/now-python/src/index.ts

```diff
@@ -42,7 +64,11 @@ export async function downloadFilesInWorkPath({
   if (meta.isDev) {
     // Old versions of the CLI don't assign this property
     const { devCacheDir = join(workPath, '.now', 'cache') } = meta;
-    const destCache = join(devCacheDir, basename(entrypoint, '.py'));
+    const entrypointCacheDir = join(
+      dirname(entrypoint),
+      basename(entrypoint, '.py')
+    ).replace(new RegExp(sep, 'g'), '__');
```

Comment:
Why use `__` here?

Reply:
No particular reason, just a delimiter that seemed unlikely to occur in the original file and directory names. Would you prefer another one?

Reply:
I mean, why was this changed? Was there a bug?

Reply:
Before it was
```ts
const destCache = join(devCacheDir, basename(entrypoint, '.py'));
```
and the problem is that there could be multiple APIs with the same entrypoint file name, e.g.:
```
api/py-func-a/index.py
api/py-func-b/index.py
```
Then, the cache directory is `{devCacheDir}/index` for both, which is a problem. Now, the full path is used as the cache directory name, in this example `{devCacheDir}/api__py-func-a__index` and `{devCacheDir}/api__py-func-b__index`, so the cache directories are unique.

Reply:
@TooTallNate What do you think. Should we follow the same pattern here as `@vercel/node`?

Reply:
Any progress here?

Reply:
@styfle @TooTallNate Do you require any further changes here?

Reply:
Why have the `.replace(new RegExp(sep, 'g'), '__')` part at all? Creating subdirectories makes sense to me, and I believe we do it like that in other Runtimes.